src/main/java/chess/domain/square/File.java

```diff
@@ -0,0 +1,67 @@
+package chess.domain.square;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public enum File {
+
+    a(1),
+    b(2),
+    c(3),
+    d(4),
+    e(5),
+    f(6),
+    g(7),
+    h(8);
```

Comment:
```suggestion
    h(8),
    ;
```
(반영하지 않으셔도 됩니다.) 추후 추가 편의성을 위해서 `,`를 뒤에 붙여두기도 해요.

Reply:
추후 추가 편의성을 고려하지는 못 했네요! 반영하였습니다.😸